tests/acceptance/features/apiSharingNg/sharedWithMeSyncDisabled.feature

```diff
@@ -2790,3 +2790,285 @@ Feature: listing sharedWithMe when auto-sync is disabled
         }
       }
       """
+
+
+  Scenario: user who is also a member of group lists file shared with them from project-space
+    Given using spaces DAV path
+    And group "grp1" has been created
+    And the following users have been added to the following groups
+      | username | groupname |
+      | Brian    | grp1      |
+    And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "projectSpace" with the default quota using the Graph API
+    And user "Alice" has uploaded a file inside space "projectSpace" with content "to share" to "textfile.txt"
+    And user "Alice" has sent the following share invitation:
+      | resource        | textfile.txt |
+      | space           | projectSpace |
+      | sharee          | grp1         |
+      | shareType       | group        |
+      | permissionsRole | File Editor  |
```

Comment:
```suggestion
      | permissionsRole | Viewer  |
```

Reply:
let's share with different role with different sharee.

Reply:
> let's share with different role with different sharee.

Does that make any sense in this scenario?

Reply:
there is a bug report on losing permissions of shares from same file (https://github.com/owncloud/ocis/issues/8464). so we can check that different permissions are in intact.